application/org.openjdk.jmc.browser/src/main/resources/org/openjdk/jmc/browser/views/messages.properties

```diff
@@ -57,7 +57,7 @@ JVMBrowserView_NO_LOCAL_JVMS_TITLE=No local JVMs detected
 JVMBrowserView_NO_LOCAL_JVMS_MESSAGE=No local (attachable) JVMs were detected, not even the JVM running this instance of JDK Mission Control.
 # JVMBrowserView_NO_LOCAL_JVMS_WARN_CAUSE has newlines placed so that the rows doesn't get too long. It would be
 # great if the translation also contains newlines so the rows get about the same length.
-JVMBrowserView_NO_LOCAL_JVMS_WARN_CAUSE=Local JVMs will not be detected if the JDK Mission Control Client is run with a JRE instead of a JDK, or if the hsperfdata_<user> folder in TMP is not writable. See the FAQ help section for more information.
+JVMBrowserView_NO_LOCAL_JVMS_WARN_CAUSE=Local JVMs will not be detected if the JDK Mission Control Client is run with a JRE instead of a JDK, if the JDK is <11, or if the hsperfdata_<user> folder in TMP is not writable. See the FAQ help section for more information.
```

Comment:
Perhaps instead spell out the less than and expand to JDK version? Like so ", if the JDK version is less than 11, "

Reply:
Ok, this one too. Saw this first comment to late -> two commits

Reply:
No problem. You can have a gazillion commits - the friendly neighborhood Skara-bot will automatically squash. :)